### PR TITLE
add item and recipe to craft yogurt

### PR DESCRIPTION
**Pull request by esarbe**
#### Summary

Content "Adds a means to craft yogurt from milk and a source of lactobacillus."

#### Purpose of change

Make yogurt craftable and available for comestion even long after the last food stores have crumbled to dust.

#### Describe the solution

The solution adds a recipe and a temporary placeholder product ("fermenting_yogurt") to simulate the fermenting process. To initiate fermentation, a source of lactobacillus is required; preferably a teaspoon of yogurt but in a pinch a fruit or stem of capsicum (e.g. chili) or even a leaf of cabbage will do.

#### Describe alternatives you've considered

Don't add a recipe and miss out on home-made yogurt.

#### Testing

Create  a new game, spawn milk, some source of lactobacillus, fire and a container for boiling. Craft fermenting yogurt, wait eight hours, harvest yogurt. Hmm, delicious.

#### Additional context

I Can't Believe It's Not Yogurt!

### CODE REVIEW

**Comment by esarbe** [comment]
Is there anything that I can do to move this along?

**Comment by BrettDong** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md

File: data/json/recipes/food/brewing.json [hunk 1]
```diff
@@ -437,5 +437,28 @@
...
+    "qualities": [ { "id": "BOIL", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
+    "components": [
+      [ [ "milk_standard_raw", 9, "LIST" ] ],
+      [
+        [ "cabbage", 1 ],
+        [ "chili_pepper", 1],
+        [ "yoghurt", 1 ]
+      ]
+    ]
```
Comment (dseguin) [inline data/json/recipes/food/brewing.json:462]:
```suggestion
    "components": [ [ [ "milk_standard_raw", 9, "LIST" ] ], [ [ "cabbage", 1 ], [ "chili_pepper", 1 ], [ "yoghurt", 1 ] ] ]
```

File: data/json/items/comestibles/brewing.json [hunk 2]
```diff
@@ -682,5 +682,35 @@
...
+    "name": { "str": "fermenting yogurt" },
+    "weight": "236 g",
+    "color": "white",
+    "spoils_in": "1 d",
+    "container": "jug_plastic",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 8,
+    "calories": 132,
+    "description": "Milk inoculated with lactobacillus. It requires some time fermenting to hopefully end up as delicious yogurt.",
```
Comment (dseguin) [inline data/json/items/comestibles/brewing.json:699]:
```suggestion
    "description": "Milk inoculated with lactobacillus.  It requires some time fermenting to hopefully end up as delicious yogurt.",
```

**Comment by esarbe** [comment]
Thanks for bringing the json in line and merging!

**Pull request merged**